### PR TITLE
adding contrib init for reports and modifying status check to have ou…

### PR DESCRIPTION
**Pull request by miloaec**
I add an init.d file for the reports and change the status check in order to print out each process status (ie web0, handler0, handler1..)

/etc/init.d/galaxy status

web0 (pid  28038) running...
handler0 (pid  28041) running...
handler1 (pid  28044) running...

### CODE REVIEW

File: contrib/galaxy.fedora-init [hunk 1]
```diff
@@ -22,9 +24,11 @@ start() {
 	case "$(id -un)" in
 		$RUN_AS)
 			eval "$cmd"
+			touch /var/lock/subsys/galaxy
```
Comment (nsoranzo) [inline contrib/galaxy.fedora-init]: It is quite possible that the `galaxy` user does not have permissions to write in `/var/lock/subsys/`.

File: contrib/galaxy.fedora-init [hunk 2]
```diff
@@ -53,19 +59,17 @@ stop() {
...
+	if [[ $(grep '\[server:' $RUN_IN/config/galaxy.ini|awk -F'(:)|(])' '{ print $2 }') == 'main' ]]
+	then
+		echo -n "$SERVICE_NAME status: "
+		status -p $RUN_IN/paster.pid galaxy
+	else
+		for proc in $(grep '\[server:' $RUN_IN/config/galaxy.ini|awk -F'(:)|(])' '{ print $2 }')
+		do
+			status -p $RUN_IN/${proc}.pid ${proc}
+		done
+	fi																			}
```
Comment (nsoranzo) [inline contrib/galaxy.fedora-init]: I'd leave an empty line here to separate the 2 functions.

File: contrib/galaxy.fedora-init (hunk 1 above)
Comment (miloaec) [inline contrib/galaxy.fedora-init]: You are right.

File: contrib/galaxy_reports.fedora-init [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+	echo -n "Stopping $SERVICE_NAME... "
+	
+	cmd="cd $RUN_IN && sh run_reports.sh --stop-daemon"
+
+	case "$(id -un)" in
+		$RUN_AS)
+			eval "$cmd"
+			;;
+		root)
+			su - $RUN_AS -c "$cmd"
```
Comment (nsoranzo) [inline contrib/galaxy_reports.fedora-init:48]: `;;`

**Pull request merged**